scss/stanford_media.multi_step.scss

```diff
@@ -15,3 +15,11 @@
     max-height: 150px;
   }
 }
+
+table .video-embed-field-lazy {
+  width: fit-content;
```

Comment:
This doesn't appear to be supported well. https://caniuse.com/#search=fit-content

Is there another option that would be supported by more browsers?

Reply:
its just in the media library so its not used anywhere on the front end. because of that, i just set a hard width value of 200px. it accomplishes the same thing